### PR TITLE
Initial commit with simple page

### PR DESCRIPTION
**Pull request by GreyYar**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,29 @@
...
+
+Тех задание:  
+Проект "Anime Sage"
+
+Requirement Stack list:  
+* Webpack;  
+* ReactJS;  
+* Lint.  
+
+Описание:
```
Comment (RCAPDART) [inline README.md]: Please don't use Russian language. Use English :)

File: index.html [hunk 2]
```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="utf-8" />
+    <title>Anime Quiz</title>
+	<link rel="stylesheet" href="stylesheet.css"/>
+</head>
+<body>
+    <div id="app">
+		<div class="background-image"></div>
```
Comment (RCAPDART) [inline index.html]: I think, that you can move all content to the ReactJS APP.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+    <script type="text/babel">
+		var name = "Анон"
+		class Hello extends React.Component {
+			render() {
+				return <h1>Привет, {name}!</h1>;
+			}
+		}
+        ReactDOM.render(
+            <Hello />,
+            document.getElementById("greeting"),
```
Comment (RCAPDART) [inline index.html]: element id in your case should be "app"

File: index.html [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+			<button>Go!</button>
+		</div>
+	</div>
+      
+    <script crossorigin src="https://unpkg.com/react@16/umd/react.production.min.js"></script>
+    <script crossorigin src="https://unpkg.com/react-dom@16/umd/react-dom.production.min.js"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/babel-standalone/6.25.0/babel.min.js"></script>
+    <script type="text/babel">
+		var name = "Анон"
+		class Hello extends React.Component {
```
Comment (RCAPDART) [inline index.html]: Please use JS/JSX file for storing this logic. Not html.

File: index.html [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+			render() {
+				return <h1>Привет, {name}!</h1>;
+			}
+		}
+        ReactDOM.render(
+            <Hello />,
```
Comment (RCAPDART) [inline index.html]: It is better to use "App" component as an entry point :)

**Review by RCAPDART — CHANGES_REQUESTED** [review]
Please see comments:

File: stylesheet.css [hunk 6]
```diff
@@ -0,0 +1,29 @@
+body {
+	background-color: DarkSlateBlue ;
+}
+
+.background-image {
+  position: fixed;
+  left: 0;
+  right: 0;
+  z-index: 1;
+  background-image: url('\zaraki.png');
```
Comment (Engwyn) [inline stylesheet.css]: Lets use something more neutral as background

File: index.html (hunk 5 above)
Comment (GreyYar) [reply index.html]: Done

File: index.html (hunk 4 above)
Comment (GreyYar) [reply index.html]: Moved to separate JSX

File: index.html (hunk 3 above)
Comment (GreyYar) [reply index.html]: Thanks, fixed

File: index.html (hunk 2 above)
Comment (GreyYar) [reply index.html]: Moved to JSX

File: stylesheet.css (hunk 6 above)
Comment (GreyYar) [reply stylesheet.css]: Changed to more neutral bg, please let me know if it doesn't fit too

File: index.html [hunk 7]
```diff
@@ -0,0 +1,16 @@
...
+    <title>Anime Sage</title>
+	<link rel="stylesheet" href="stylesheet.css"/>
+</head>
+<body>
+    <div id="app"></div>
+    <!--to be replaced with webpack:-->
+	<script src="https://unpkg.com/react@16/umd/react.development.js" crossorigin></script>
+	<script src="https://unpkg.com/react-dom@16/umd/react-dom.development.js" crossorigin></script>
+	<script src="https://unpkg.com/babel-standalone@6/babel.min.js"></script>
+    <script type="text/babel" src="index.jsx"></script>
```
Comment (RCAPDART) [inline index.html:14]: Please check tabulation

**Review by RCAPDART — APPROVED** [review]

**Review by Engwyn — APPROVED** [review]

**Pull request merged**